core/tests/ledger_cleanup.rs

```diff
@@ -234,7 +235,8 @@ mod tests {
         solana_logger::setup_with("error,ledger_cleanup::tests=info");
 
         let ledger_path = get_tmp_ledger_path!();
-        let mut blockstore = Blockstore::open(&ledger_path).unwrap();
+        let mut blockstore =
+            Blockstore::open_with_options(&ledger_path, BlockstoreOptions::default()).unwrap();
```

Comment:
I guess this is changed over to use `open_with_options()` instead of `open()` for a subsequent PR ?